### ISSUE TITLE
Icons

### ISSUE THREAD

**Issue by goodmind** [arturbien/React95#29]
Is there icon pack with original icons?

**Comment by arturbien** [comment]
@goodmind I found some icon sets here:
https://artage.io/en/icon-packs/original-windows-95-icons
http://www.iconwanted.com/en/ipapun/original-windows-95.html
https://win98icons.alexmeub.com/

We could find some more and add it to the library or as an additional package. We would have to make sure all of the icons are the same size and aligned properly (although I think that when we find a resource with original .ico icons, sizing and alignment should be alright).

**Comment by maracuja-juice** [comment]
You need to be careful about the license of these. Do you know what the license is of these icons? (couldn't find it out)

**Comment by arturbien** [comment]
Yeah that might be a problem. Well, maybe it's time to find some designers to make some custom icons 🤷‍♂️  or modify the OG ones sufficiently enough 😂

**Comment by mandrilla34** [comment]
Hello guys, there is a package with the same React 95 theme and they provide icons ready to impose: [Link package icons](https://github.com/React95/React95/tree/master/packages/icons)
